clients/osv/src/test/assets/vulnerability/examples/7.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Change indentation of file to 2 spaces. This is what seems to used in ORT based on search for JSON files, see for example 

- https://github.com/oss-review-toolkit/ort/blob/main/integrations/schemas/package-configuration-schema.json
- https://github.com/oss-review-toolkit/ort/blob/main/scanner/src/test/assets/scancode-with-issues.json

Reply:
I deliberately choose to keep these files in the original state, so that it's easy to update them in the future and to see that they are identical with the original ones. Fine?